PTC/Controllers/BaseController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using PTCData;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+
+namespace PTC.Controllers
+{
+    public abstract class BaseController : Controller
+    {
+        protected BaseController(ITrainingProductManager tpmgr)
+        {
+            _tpmanager = tpmgr;
```

Comment:
please write out toiletpaper instead of tp

Reply:
No can do Beavis..